sympy/algebras/quaternion.py

```diff
@@ -547,16 +547,19 @@ def rotate_point(pin, r):
         ==========
 
         pin : tuple
-            A 3-element tuple of coordinates of a point. This point will be
-            the axis of rotation.
-        r
-            Angle to be rotated.
+            A 3-element tuple of coordinates of a point which needs to be
+            rotated.
+        r : Quaternion or tuple
+            Axis of rotation.
```

Comment:
A quaternion will determine both the axis and the angle of rotation. Maybe this could be "Axis and angle of rotation".

Reply:
Ok, I'll change it